KmsClient.js

```diff
@@ -676,6 +791,36 @@ export class KmsClient {
   }
 }
 
+/**
+ * @param {object} options - Options hashmap.
+ * @param {string} options.message - Error message.
+ * @param {Error} options.cause - Source error for wrapping.
+ * @param {string} [options.notFoundMessage] - Optional 'not found' message.
+ */
+function _handleClientError({
+  message, cause, notFoundMessage = 'Key not found'
+}) {
+  let error;
+  if(cause.status === 409) {
+    error = new Error(
+      `Duplicate error during WebKMS client operation: ${message}`);
+    error.name = 'DuplicateError';
```

Comment:
Are we sure that this is the only time a 409 is thrown? Just because there were some particular 409 errors we handled individually above doesn't mean that a 409 couldn't have happened in one of the other calls. If so, this message will not be correct when we get here. I think we should keep the individual 409 (and other error code) handling back where it was before.

Reply:
@dlongley - not sure I fully understand. A 409 is always a duplicate error, no? And regardless of where it was thrown, it's specifically in the context of a WebKMS client operation.

Reply:
No, it's an invalid state error which could be due to, for example, a duplicate error or a conflict error. A WebKMS client operation could throw a duplicate error or a conflict error (when sequence numbers for configs are not in order). So we know there are at least two different cases and now they've been (mis)folded into one.

Reply:
@dlongley - Reverted to previous 409 logic (only marking as duplicate error in the original two cases).